packages/docusaurus-theme-classic/src/theme/DocBreadcrumbs/index.tsx

```diff
@@ -49,11 +54,14 @@ function BreadcrumbsItem({
 }
 
 function HomeBreadcrumbItem() {
-  const homeHref = useBaseUrl('/');
   return (
-    <BreadcrumbsItem>
-      <BreadcrumbsItemLink href={homeHref}>🏠</BreadcrumbsItemLink>
-    </BreadcrumbsItem>
+    <li className="breadcrumbs__item">
```

Comment:
First: do we really want the home to not be part of the breadcrumbs? What is the best practice in this regard?

---

This is exactly something I'd like avoid: using Infima class strings on higher-level concepts.

If you want to use Infima, use the React-Infima component `<BreadcrumbsItem>`

If it's not good enough for this use case, then add props and flexibility to make it work.

Think this way:
- infima-react: design system, can use infima classes
- theme-infima: can use infima-react but is not allowed to use infima classes as strings

This component is part of theme-infima, probably not infima-react design-system because it's very specific to this use-case: we might want to display breadcrumbs in otherplace with this home link

Reply:
To make it clearer, outside of your design system, the code of your app should be very high-level, clear and look that way:

https://kentcdodds.com/blog/compound-components-with-react-hooks

<img width="519" alt="image" src="https://user-images.githubusercontent.com/749374/154479285-f95a8c71-e9ed-4909-9de5-ad20d957a090.png">

We should strive to not mix low/high-level details

Reply:
So as of now, `HomeBreadcrumbItem` and `BreadcrumbsItem` have the same level of abstraction. I think that's good enough.